src/runtime/Makefile

```diff
@@ -439,7 +453,7 @@ USER_VARS += DEFDISABLEBLOCK
 USER_VARS += DEFBLOCKSTORAGEDRIVER_ACRN
 USER_VARS += DEFBLOCKSTORAGEDRIVER_FC
 USER_VARS += DEFBLOCKSTORAGEDRIVER_QEMU
-USER_VARS += DEFSHAREDFS_QEMU_VIRTIOFS
```

Comment:
This change may break distro builds that rely on the old name. But I agree it's an improvement. I would just put it in some separate commit, because it's not obviously related to s390.

Reply:
Done!